src/gui/socketapi.cpp

```diff
@@ -731,10 +1005,14 @@ void SocketApi::sendSharingContextMenuOptions(const FileData &fileData, SocketLi
 
     // If sharing is globally disabled, do not show any sharing entries.
     // If there is no permission to share for this file, add a disabled entry saying so
+
+    listener->sendMessage(QLatin1String("MENU_ITEM:OFFLINE_DOWNLOAD_MODE") + flagString + tr("0ffline"));
```

Comment:
Offline not 0ffline I guess. ;-)

Reply:
?

Reply:
Look at the translated string, it has a zero instead of an uppercase o